app/services/pipeline_report_service.rb

```diff
@@ -67,13 +67,39 @@ def call
   end
 
   def generate
-    pipeline_run = PipelineRun.find(@pipeline_run_id)
     if @pipeline_run_id.nil?
-      Rails.logger.error("Pipeline run doesn't exist.")
-      return ""
-    elsif pipeline_run.total_reads.nil? || pipeline_run.adjusted_remaining_reads.nil?
-      Rails.logger.error("Pipeline run #{@pipeline_run_id} has no reads.")
-      return ""
+      return JSON.dump(
+        pipelineRunInfo: {
+          pipelineRunComplete: false,
+          pipelineRunFailed: false,
+          sampleStatus: "Waiting to Start or Receive Files",
+          errorMessage: nil,
+          knownUserError: nil,
+        },
+        metadata: {},
+        counts: {},
```

Comment:
Do we need to return these keys?

Reply:
`SampleViewV2` attempts to parse/iterate through these keys, so the frontend will throw some errors if they don't exist. I could alternatively add some checks to `fetchSampleReportData` to avoid those errors. Which way is preferred?

Reply:
I would prefer that the frontend adjusts.

Reply:
Done.